### ISSUE TITLE
Claims over time component settings cause 500 error

### ISSUE THREAD

**Issue by Uafinder** [Kademi/kademi-dev#4767]
![image](https://user-images.githubusercontent.com/1945230/34779794-e9bc5bb8-f629-11e7-8e43-7a7023298ed1.png)
```
04:17:39 ERROR io.milton.http.http11.GetHandler  - Exception sending content for:/_components/claimsOverTime of resource type: io.milton.cloud.server.web.EditorSnippetHandlerResource.ComponentSnippetResource
04:17:39 java.lang.RuntimeException: Error locating resource: vladmoblixclone-vladtest20f.kademi-ci.co:/theme/apps/salesDataClaimer/components/claimsOverTimeSettings.html Exception: Unable to find resource 'vladmoblixclone-vladtest20f.kademi-ci.co:/theme/apps/salesDataClaimer/components/claimsOverTimeSettings.html'
04:17:39 	at io.milton.cloud.server.web.templating.TextTemplater.writePage(TextTemplater.java:201)
04:17:39 	at io.milton.cloud.server.apps.ComponentBean.renderSettings(ComponentBean.java:246)
04:17:39 	at io.milton.cloud.server.web.EditorSnippetHandlerResource$ComponentSnippetResource.sendContent(EditorSnippetHandlerResource.java:345)
04:17:39 	at io.milton.http.CompressingResponseHandler.respondContent(CompressingResponseHandler.java:116)
04:17:39 	at io.milton.http.AbstractWrappingResponseHandler.respondContent(AbstractWrappingResponseHandler.java:90)
04:17:39 	at io.milton.cloud.server.web.KademiResponseHandler.respondContent(KademiResponseHandler.java:38)
04:17:39 	at io.milton.http.http11.GetHandler.sendContent(GetHandler.java:223)
04:17:39 	at io.milton.http.http11.GetHandler.processExistingResource(GetHandler.java:85)
04:17:39 	at io.milton.http.ResourceHandlerHelper.processResource(ResourceHandlerHelper.java:196)
04:17:39 	at io.milton.http.http11.GetHandler.processResource(GetHandler.java:66)
04:17:39 	at io.milton.http.ResourceHandlerHelper.process(ResourceHandlerHelper.java:127)
04:17:39 	at io.milton.http.http11.GetHandler.process(GetHandler.java:60)
04:17:39 	at io.milton.http.StandardFilter.process(StandardFilter.java:50)
04:17:39 	at io.milton.http.FilterChain.process(FilterChain.java:46)
04:17:39 	at co.kademi.server.subscription.KademiSubscriptionFilter.process(KademiSubscriptionFilter.java:45)
04:17:39 	at io.milton.http.FilterChain.process(FilterChain.java:46)
04:17:39 	at co.kademi.server.idp.IDPFilter.process(IDPFilter.java:41)
04:17:39 	at io.milton.http.FilterChain.process(FilterChain.java:46)
04:17:39 	at io.milton.cloud.server.db.utils.MiltonOpenSessionInViewFilter$1.execute(MiltonOpenSessionInViewFilter.java:41)
04:17:39 	at io.milton.context.RootContext.execute(RootContext.java:90)
04:17:39 	at io.milton.cloud.server.db.utils.MiltonOpenSessionInViewFilter.process(MiltonOpenSessionInViewFilter.java:34)
04:17:39 	at io.milton.http.FilterChain.process(FilterChain.java:46)
04:17:39 	at io.milton.http.HttpManager.process(HttpManager.java:158)
04:17:39 	at co.kademi.server.Kademi$1.service(Kademi.java:191)
04:17:39 	at org.glassfish.grizzly.http.server.HttpHandler$1.run(HttpHandler.java:224)
04:17:39 	at org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.doWork(AbstractThreadPool.java:591)
04:17:39 	at org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.run(AbstractThreadPool.java:571)
04:17:39 	at java.lang.Thread.run(Thread.java:748)
04:17:39 Caused by: org.apache.velocity.exception.ResourceNotFoundException: Unable to find resource 'vladmoblixclone-vladtest20f.kademi-ci.co:/theme/apps/salesDataClaimer/components/claimsOverTimeSettings.html'
04:17:39 	at org.apache.velocity.runtime.resource.ResourceManagerImpl.loadResource(ResourceManagerImpl.java:474)
04:17:39 	at org.apache.velocity.runtime.resource.ResourceManagerImpl.getResource(ResourceManagerImpl.java:352)
04:17:39 	at org.apache.velocity.runtime.RuntimeInstance.getTemplate(RuntimeInstance.java:1566)
04:17:39 	at org.apache.velocity.runtime.RuntimeInstance.getTemplate(RuntimeInstance.java:1546)
04:17:39 	at org.apache.velocity.app.VelocityEngine.getTemplate(VelocityEngine.java:373)
04:17:39 	at io.milton.cloud.server.web.templating.TextTemplater.writePage(TextTemplater.java:199)
04:17:39 	... 27 more
04:17:39 ERROR io.milton.http.StandardFilter  - exception sending content
04:17:39 java.lang.RuntimeException: java.lang.RuntimeException: Error locating resource: vladmoblixclone-vladtest20f.kademi-ci.co:/theme/apps/salesDataClaimer/components/claimsOverTimeSettings.html Exception: Unable to find resource 'vladmoblixclone-vladtest20f.kademi-ci.co:/theme/apps/salesDataClaimer/components/claimsOverTimeSettings.html'
04:17:39 	at io.milton.http.http11.GetHandler.sendContent(GetHandler.java:233)
04:17:39 	at io.milton.http.http11.GetHandler.processExistingResource(GetHandler.java:85)
04:17:39 	at io.milton.http.ResourceHandlerHelper.processResource(ResourceHandlerHelper.java:196)
04:17:39 	at io.milton.http.http11.GetHandler.processResource(GetHandler.java:66)
04:17:39 	at io.milton.http.ResourceHandlerHelper.process(ResourceHandlerHelper.java:127)
04:17:39 	at io.milton.http.http11.GetHandler.process(GetHandler.java:60)
04:17:39 	at io.milton.http.StandardFilter.process(StandardFilter.java:50)
04:17:39 	at io.milton.http.FilterChain.process(FilterChain.java:46)
04:17:39 	at co.kademi.server.subscription.KademiSubscriptionFilter.process(KademiSubscriptionFilter.java:45)
04:17:39 	at io.milton.http.FilterChain.process(FilterChain.java:46)
04:17:39 	at co.kademi.server.idp.IDPFilter.process(IDPFilter.java:41)
04:17:39 	at io.milton.http.FilterChain.process(FilterChain.java:46)
04:17:39 	at io.milton.cloud.server.db.utils.MiltonOpenSessionInViewFilter$1.execute(MiltonOpenSessionInViewFilter.java:41)
04:17:39 	at io.milton.context.RootContext.execute(RootContext.java:90)
04:17:39 	at io.milton.cloud.server.db.utils.MiltonOpenSessionInViewFilter.process(MiltonOpenSessionInViewFilter.java:34)
04:17:39 	at io.milton.http.FilterChain.process(FilterChain.java:46)
04:17:39 	at io.milton.http.HttpManager.process(HttpManager.java:158)
04:17:39 	at co.kademi.server.Kademi$1.service(Kademi.java:191)
04:17:39 	at org.glassfish.grizzly.http.server.HttpHandler$1.run(HttpHandler.java:224)
04:17:39 	at org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.doWork(AbstractThreadPool.java:591)
04:17:39 	at org.glassfish.grizzly.threadpool.AbstractThreadPool$Worker.run(AbstractThreadPool.java:571)
04:17:39 	at java.lang.Thread.run(Thread.java:748)
04:17:39 Caused by: java.lang.RuntimeException: Error locating resource: vladmoblixclone-vladtest20f.kademi-ci.co:/theme/apps/salesDataClaimer/components/claimsOverTimeSettings.html Exception: Unable to find resource 'vladmoblixclone-vladtest20f.kademi-ci.co:/theme/apps/salesDataClaimer/components/claimsOverTimeSettings.html'
04:17:39 	at io.milton.cloud.server.web.templating.TextTemplater.writePage(TextTemplater.java:201)
04:17:39 	at io.milton.cloud.server.apps.ComponentBean.renderSettings(ComponentBean.java:246)
04:17:39 	at io.milton.cloud.server.web.EditorSnippetHandlerResource$ComponentSnippetResource.sendContent(EditorSnippetHandlerResource.java:345)
04:17:39 	at io.milton.http.CompressingResponseHandler.respondContent(CompressingResponseHandler.java:116)
04:17:39 	at io.milton.http.AbstractWrappingResponseHandler.respondContent(AbstractWrappingResponseHandler.java:90)
04:17:39 	at io.milton.cloud.server.web.KademiResponseHandler.respondContent(KademiResponseHandler.java:38)
04:17:39 	at io.milton.http.http11.GetHandler.sendContent(GetHandler.java:223)
04:17:39 	... 21 more
04:17:39 Caused by: org.apache.velocity.exception.ResourceNotFoundException: Unable to find resource 'vladmoblixclone-vladtest20f.kademi-ci.co:/theme/apps/salesDataClaimer/components/claimsOverTimeSettings.html'
04:17:39 	at org.apache.velocity.runtime.resource.ResourceManagerImpl.loadResource(ResourceManagerImpl.java:474)
04:17:39 	at org.apache.velocity.runtime.resource.ResourceManagerImpl.getResource(ResourceManagerImpl.java:352)
04:17:39 	at org.apache.velocity.runtime.RuntimeInstance.getTemplate(RuntimeInstance.java:1566)
04:17:39 	at org.apache.velocity.runtime.RuntimeInstance.getTemplate(RuntimeInstance.java:1546)
04:17:39 	at org.apache.velocity.app.VelocityEngine.getTemplate(VelocityEngine.java:373)
04:17:39 	at io.milton.cloud.server.web.templating.TextTemplater.writePage(TextTemplater.java:199)
04:17:39 	... 27 more
```

1. Add component to the custom page
2. Push settings icon on it.
http://vladtest20f.admin.kademi-ci.co/pages/vladmoblixclone/version1/
http://vladtest20f.admin.kademi-ci.co/pages/vladmoblixclone/version1/?contenteditor=sales4.html

**Comment by bradmac** [comment]
Cause is:
```
Unable to find resource 'vladmoblixclone-vladtest20f.kademi-ci.co:/theme/apps/salesDataClaimer/components/claimsOverTimeSettings.html'
```

**Comment by bradmac** [comment]
So possibly a lib hasnt been installed. @ducdhm please check dependencies.

**Comment by bradmac** [comment]
@gonzalon please check this out

**Comment by gonzalon** [comment]
Fixed in salesDataClaimer: 1.1.8

**Comment by Uafinder** [comment]
it works!
